resources/src/androidMain/kotlin/dev/icerock/moko/resources/ColorResourceExt.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.resources
+
+import android.content.Context
+import android.content.res.Configuration
+import dev.icerock.moko.graphics.Color
+
+fun ColorResource.getColor(context: Context): Color {
```

Comment:
i think this extension required for `ColorResource.Themed` only

Reply:
Why? With this function an user does't need to know what color will be obtained from the resource. So you can apply a color without unnecessary checks.

Reply:
agree with you